src/app/containers/PodcastPromo/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import PodcastPromo from '@bbc/psammead-podcast-promo';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import ImageWithPlaceholder from '#containers/ImageWithPlaceholder';
+
+const getSrcFromSize = (url, size) => {
```

Comment:
definitely no utiliity function available that does this?

Reply:
Not so sure. There are a couple of utility functions that do something similar, but require constraints that are not available here: e.g. we don't fetch our images from news.

I think the srcset refactor should be a different issue that consolidates our use of srcset